src/statisticBlock.js

```diff
@@ -97,69 +98,103 @@ class StatisticBlock extends Component {
       }
     }
   }
-
   checkRequiredSize(){
     let element = this.props.element;
-
     let scrollWidth = element.scrollWidth * 0.95;
     let scrollHeight = element.scrollHeight * 0.95;
 
     if(this.props.options.autoSize) {
```

Comment:
[optional] If you flip the if statement here it's easier to see through what happening